spr.lua

```diff
@@ -308,6 +308,18 @@ local typeMetadata = {
 			return Vector3.new(value[1], value[2], value[3])
 		end,
 	},
+	
+	CFrame = {
+		springType = LinearSpring.new,
+
+		toIntermediate = function(value)
+			return {value:GetComponents()}
+		end,
+
+		fromIntermediate = function(value)
+			return CFrame.new(unpack(value))
```

Comment:
```suggestion
			return CFrame.new(unpack(value)):Orthonormalize()
```

We probably could orthonormalize the CFrame before returning it back to the instance. Because each component is springed linearly without regard for the other ones, it's possible that we end up with a "bad" CFrame, that when applied on something like a camera can have some funny effects.